packages/uauth/src/index.ts

```diff
@@ -0,0 +1,336 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+import type { StaticJsonRpcProvider as StaticJsonRpcProviderType } from '@ethersproject/providers'
+import UAuth from '@uauth/js'
```

Comment:
Can this be imported dynamically within the function?

Reply:
Moved this to dynamic under both the getUauthUser function as well as the uauth function